.github/workflows/manual_release.yml

```diff
@@ -12,7 +12,34 @@ on:
         type: string
         required: false
 
+permissions:
+  contents: write
+  id-token: "write"
+
 jobs:
+  get-commit-sha:
+    name: Get Commit Sha
+    runs-on: ubuntu-latest
+    outputs:
+      short-sha: ${{ steps.get-short-sha.outputs.short-sha }}
+      full-sha: ${{ steps.get-full-sha.outputs.full-sha }}
+    steps:
+      - name: Add SHORT_SHA env property with commit short sha
+        id: get-short-sha
+        run: |
+          if [ -z "${{ inputs.OVERRIDE_COMMIT }}" ]; then
+            USED_SHA=${{ github.sha }}
+          else
+            USED_SHA=${{ inputs.OVERRIDE_COMMIT }}
+          fi
+          echo "short-sha=$(echo $USED_SHA | cut -c1-7)" >> $GITHUB_OUTPUT
```

Comment:
This talks a bout a `SHORT_SHA` env, making it sounds like an env-var, but it is set as `short-sha` and used like that everywhere afaict, making this comment a bit confusing to me